bindings/java/src/test/com/apple/foundationdb/test/TuplePerformanceTest.java

```diff
@@ -292,7 +302,7 @@ public void run() {
 	}
 
 	public static void main(String[] args) {
-		TuplePerformanceTest tester = new TuplePerformanceTest(new Random(), 100_000, 10_000_000, GeneratedTypes.ALL);
+		TuplePerformanceTest tester = new TuplePerformanceTest(new Random(), 100_000, 10_000_000, GeneratedTypes.STRING_LIKE);
```

Comment:
Was it intended for this change to be permanent?

Reply:
Oops, no